cws-core/src/main/java/jpl/cws/core/db/SchedulerDbService.java

```diff
@@ -48,17 +48,19 @@ public class SchedulerDbService extends DbService implements InitializingBean {
 	public static final int DEFAULT_WORKER_PROC_DEF_MAX_INSTANCES = 1;
 	public static final int PROCESSES_PAGE_SIZE = 100;
 	
-	// KEY FOR THIS IS:  KEY `claimKey` (`status`,`proc_def_key`,`priority`,`created_time`)
+
 	public static final String FIND_CLAIMABLE_ROWS_SQL = 
 			"SELECT uuid FROM cws_sched_worker_proc_inst " +
 			"WHERE " +
 			"  status='"+PENDING+"' AND " +
-			"  proc_def_key=? " +
+			"  proc_def_key=?" +
```

Comment:
this line will cause an issue.  There needs to be a space

Reply:
Fixed: https://github.com/NASA-AMMOS/common-workflow-service/pull/126/commits/0f4d595ec8a826d42ec4f0ca3462ba93342cc193